viewer/src/tools/Cognite3DViewerToolBase.ts

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { assertNever } from '../utilities/assertNever';
+import { Cognite3DViewerTool, DisposedDelegate } from '../public/migration/Cognite3DViewerTool';
+
+/**
+ * Base class for tools attaching to a {@see Cognite3DViewer}.
+ * @internal
+ */
+export abstract class Cognite3DViewerToolBase implements Cognite3DViewerTool {
```

Comment:
This just looks off, to me. base is implementing not base? Shouldn't it be` Cognite3DViewerTool implements Cognite3DViewToolBase`?

Reply:
Interface removed